src/transaction.ts

```diff
@@ -86,6 +86,7 @@ export interface ExecuteSqlRequest extends Statement, RequestOptions {
   queryOptions?: IQueryOptions;
   requestOptions?: Omit<IRequestOptions, 'transactionTag'>;
   dataBoostEnabled?: boolean | null;
+  directedReadOptions?: google.spanner.v1.IDirectedReadOptions;
```

Comment:
just checking if customers can still pass `null` here right?

Reply:
We have '?' which marks the property optional